roles/openshift_node_certificates/meta/main.yml

```diff
@@ -13,4 +13,4 @@ galaxy_info:
   - cloud
   - system
 dependencies:
-- role: openshift_ca
+- role: openshift_facts
```

Comment:
Absolutely.

On Wednesday, July 20, 2016, Jason DeTiberus <notifications@github.com>
wrote:

> In roles/openshift_node_certificates/meta/main.yml
> <https://github.com/openshift/openshift-ansible/pull/1990#discussion_r71633756>
> :
>
> > @@ -13,4 +13,4 @@ galaxy_info:
> >    - cloud
> >    - system
> >  dependencies:
> > -- role: openshift_ca
> > +- role: openshift_facts
>
> Should we add some type of a guard here in the off chance that someone
> tries to apply this role before the CA has been created on the ca host?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/1990/files/2ebd7ec5ec028ced2c728d8dd6766a9149333027..ea979f5a4790dcbf6363f4e0c7fb0a8e4bff7540#r71633756>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAC3mzs7KHqEpQG8x9mU2f_MOEehcw-uks5qXsccgaJpZM4IvNsh>
> .
>